src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
```

Comment:
For better responsiveness, it's better to use `width: 100%; max-width: 200px;` if you need to limit it.